Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -74,7 +74,8 @@ struct NoMagicNumbersRule: OptInRule {
             Example("let foo = 1 >> 2"),
             Example("let foo = 2 >> 2"),
             Example("let foo = 2 << 2"),
-            Example("let a = b / 100.0")
+            Example("let a = b / 100.0"),
+            Example("let (httpStatusCodeErrorLowerBound, httpStatusCodeErrorUpperBound) = (400, 599)")
```

Comment:
What about the case `let a = (2, 3)`? Is it not supposed to be ignored by the rule?

Reply:
So I wasn't intending to cover that case, but it would be easy to. Do you think we should?

Reply:
I think so. `let (a, b) = (2, 3)` is only a special case of `let a = (2, 3)`. But with regards to magic numbers it should not make a difference.